apps/tutorial_servo_control/src/header.h

```diff
@@ -0,0 +1,51 @@
+#ifndef TUTORIAL_SERVO_H
```

Comment:
#pragma once is preferred to using the #ifndef guards. I know that the training said to do it this way, but we now use #pragma once. The latest training reflects this.